### ISSUE TITLE
[Python] Implement new Concept Exercise: conditionals

### ISSUE THREAD

**Issue by DavidGerva** [exercism/v3#1103]
This issue describes how to implement the `conditionals` concept exercise for the Python track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student what is a `conditional` and how they are used in Python.

## Learning objectives

- create a `conditional` structure to choose something, take a decision
- use an `if...else` structure
- use an `if..elif...else` structure

## Out of scope

- ternary operators

## Concepts

- `conditionals`
- `if`
- `elif`
- `else`

## Prerequisites

- `basics`
- `booleans`
- `comparisons`

## Resources to refer to

- [The if statement](https://docs.python.org/3/reference/compound_stmts.html#the-if-statement)

### Hints

- TBD?

### After

Nothing

## Representer

No changes required.

## Analyzer

No changes required.

## Implementing

Tests should be written using unittest.TestCase and the test file named `conditionals_test.py`.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue, or ask in the `#maintaining-python` Slack channel.

## Edits
- edited for updated naming by @yawpitch
- edited for prerequisites and learning objectives detail by @BethanyG

**Comment by sachsom95** [comment]
Can I take up the issue?

**Comment by BethanyG** [comment]
_Most certainly!_
